config/buildspec.yml

```diff
@@ -21,8 +27,8 @@ phases:
         - sudo apt-get install unzip -qq -o=Dpkg::Use-Pty=0
         - cd $CODEBUILD_SRC_DIR  && chmod +x config/protoc_downloader.sh && ./config/protoc_downloader.sh
         - pip install --upgrade pip==19.3.1
-        - pip install -q pytest wheel pyYaml pytest-html keras==2.3.1 tensorflow==1.15.0 mxnet torch torchvision xgboost pre-commit tensorflow_datasets
-        - pip uninstall -y boto3 botocore
+        - if [ "$run_pytest_xgboost" = "enable" ]; then pip install --upgrade pyYaml==5.1; else pip install -q pyYaml; fi
```

Comment:
why special for xgboost?

Reply:
@anirudhacharya

Reply:
the xgboost training container has an old pyYaml version. `yaml.FullLoader` is available only after version 5.1